src/mod/module.h

```diff
@@ -528,6 +528,7 @@ typedef void (*chanout_butfunc)(int, int, const char *, ...) ATTRIBUTE_FORMAT(pr
 /* 324 - 327 */
 #define find_member_from_nick ((memberlist * (*) (char *))global[324])
 #define get_user_from_member ((struct userrec * (*) (memberlist *))global[325])
+#define splitcn ((void (*)(char *, char *, char, size_t))global[326])
```

Comment:
is one of these actually const char ptr as the input?

Reply:
none of the char * params. the 3rd paramter char could be const. but i dont see any benefit in
splitcn ((void (*)(char *, char *, const char, size_t)
do you?